mitodl/mit-open
# mitxonline etl v2 api

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/4484



### Description (What does it do?)
Switches mitx online to use v2 urls which include pagination


### How can this be tested?
1. Checkout this branch
2. make sure you have set the new v2 urls in .env:
  ```
  MITX_ONLINE_COURSES_API_URL=https://mitxonline.mit.edu/api/v2/courses/ 
  MITX_ONLINE_PROGRAMS_API_URL=https://mitxonline.mit.edu/api/v2/programs/
  ```
3. run  ```python manage.py backpopulate_mitxonline_data``` and verify it runs without issues and loads around 99 courses


### Additional Context
Sibling PR in infrastructure: https://github.com/mitodl/ol-infrastructure/pull/2476
